app/controllers/lookup_keys_controller.rb

```diff
@@ -34,4 +33,8 @@ def destroy
   def controller_permission
     'external_variables'
   end
+
+  def resource
```

Comment:
apparently there is something similar in V1::BaseController, anyway, I removed this method.